### PR TITLE
LVM-480 #comment made add/remove buttons of tutors and students consi…

### PR DESCRIPTION
**Pull request by avnighael**
Made Add/remove buttons of tutors#show view and Set tutor button of students#show view consistent with other buttons.

### CODE REVIEW

File: app/views/tutors/show.html.erb [hunk 1]
```diff
@@ -250,7 +252,9 @@
                           <td> <%= link_to s.name, s %> </td>
                           <td> <%= s.email %> </td>
                           <td> <%= form_tag('remove_student', method: 'patch') do %>
-                            <%= submit_tag('Remove Student') %>
+                            <%= button_tag(type: "submit", class: "btn btn-default") do %>
```
Comment (nabeel-) [inline app/views/tutors/show.html.erb]: Let's stick to the convention of using `btn-danger` for destructive actions like removing a student.

File: app/views/students/show.html.erb [hunk 2]
```diff
@@ -198,7 +198,9 @@
                   <% if !@tutor_options.nil? %>
                     <%= form_tag('set_tutor', method: 'put') do %>
                       <%= select_tag(:tutor_id, options_for_select(@tutor_options)) %>
-                      <%= submit_tag('Set Tutor') %>
+                      <%= button_tag(type: "submit", class: "btn btn-default") do %>
+                          Set Tutor <i class="icon-repeat"></i>
```
Comment (nabeel-) [inline app/views/students/show.html.erb]: We use [Bootstrap's glyphicons](https://getbootstrap.com/docs/3.3/components/#glyphicons) for icons. To add an icon use `<i class="glyphicon glyphicon-whatever"></i>`

Let's also try to use icons that help describe the action. For example, `glyphicon-remove` for deletions, `glyphicon-plus` for additions, etc

File: app/views/tutors/show.html.erb [hunk 3]
```diff
@@ -225,7 +225,9 @@
                   <% if !@student_options.empty? %>
                     <%= form_tag('add_student', method: 'put') do %>
                       <%= select_tag(:student_id, options_for_select(@student_options)) %>
-                      <%= submit_tag('Add Student') %>
+                      <%= button_tag(type: "submit", class: "btn btn-default") do %>
+                          Add Student <i class="icon-repeat"></i>
```
Comment (nabeel-) [inline app/views/tutors/show.html.erb]: See comment above about icons

File: app/views/tutors/show.html.erb [hunk 4]
```diff
@@ -250,7 +252,9 @@
                           <td> <%= link_to s.name, s %> </td>
                           <td> <%= s.email %> </td>
                           <td> <%= form_tag('remove_student', method: 'patch') do %>
-                            <%= submit_tag('Remove Student') %>
+                            <%= button_tag(type: "submit", class: "btn btn-default") do %>
+                                Remove Student <i class="icon-repeat"></i>
```
Comment (nabeel-) [inline app/views/tutors/show.html.erb]: See above comment about icons

**Review by nabeel- — CHANGES_REQUESTED** [review]
Looking good, just a couple of tweaks!

**Comment by kmcarol26** [comment]
No issues. Merging into master.

**Pull request merged**